src/raftstore/store/worker/region.rs

```diff
@@ -254,7 +260,9 @@ impl SnapContext {
             escape(&start_key),
             escape(&end_key)
         );
-        if let Err(e) = util::delete_all_in_range(&self.kv_db, &start_key, &end_key) {
+        if let Err(e) =
+            util::delete_all_in_range(&self.kv_db, &start_key, &end_key, self.use_delete_range)
```

Comment:
I don't think it's appropriate to use delete range here. For long running queries, delete all sst files directly may introduce wrong result.

Reply:
delete range will not break snapshot, so it is ok.

Reply:
Why? When SST files are deleted, keys in the bottom level can be seen by the snapshot.

Reply:
delete range never drop sst files directly.

Reply:
Are you misunderstand `delete files in range` and `delete range`.

Reply:
Oh, I see. I thought `use_delete_range` is a hint for `delete_file_in_range`. It's OK to use `delete_range` here. Thanks for the explanation.